api/src/main/java/io/opentelemetry/common/KeyValueConsumer.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.common;
+
+/**
+ * Used for iterating over the key-value pairs contained by an {@link ImmutableKeyValuePairs}
```

Comment:
ImmutableKeyValuePairs isn't in public API so can't link it. Just some generic word like container should be fine.

Reply:
fair enough. will update.

Reply:
done